subsys/bluetooth/host/classic/Kconfig

```diff
@@ -508,6 +508,11 @@ config BT_COD
 	  consult the following link:
 	  https://www.bluetooth.com/specifications/assigned-numbers
 
+config BT_DEFAULT_ROLE_SWITCHABLE
+	bool "Default Bluetooth Role Switch Enable/Disable State"
+	help
+	  This option sets the controller's default link policy to enable/disable the role switch.
```

Comment:
Please use similar line width as for help texts for other options, like `BT_COD` above.